src/user_defines.txt

```diff
@@ -83,9 +83,6 @@
 # Startup plays you own custom tune and crsf connect/disconnct do beep-boop
 #-DMY_STARTUP_MELODY="B5 16 P16 B5 16 P16 B5 16 P16 B5 2 G5 2 A5 2 B5 8 P4 A5 8 B5 1|140|-3"
 
-#Comment this to disable diversity function
-#-DUSE_DIVERSITY
```

Comment:
Im not sure about removing this because we still have a single DIY target which can use diversity.